src/inspector/view/qml/MuseScore/Inspector/notation/sectionbreaks/SectionBreakSettings.qml

```diff
@@ -89,4 +89,21 @@ Column {
         navigation.panel: root.navigationPanel
         navigation.row: shouldResetBarNums.navigation.row + 1
     }
+
+    PropertyCheckBox {
+        id: showCourtesySignatures
+        text: qsTrc("inspector", "Hide courtesy clefs and signatures")
```

Comment:
Can't we have that a bit more verbose, like `"Hide courtesy clefs, key and time signatures"`?

Reply:
I think plural, as multiple clefs / signatures use to be in score (each staff one).

Plural is also in horizontal frame properties.
![Hframe-Properties](https://github.com/user-attachments/assets/21c544ba-13eb-4764-afc3-647e50b300dc)

Reply:
Yes, plural